katka/viewsets.py

```diff
@@ -34,3 +34,26 @@ def destroy(self, request, *args, **kwargs):
             instance.save()
 
         return Response(status=status.HTTP_204_NO_CONTENT)
+
+
+class FilterViewMixin:
+    """
+    Uses the Serializer fields to construct GET Parameter filtering
+    """
+    def get_queryset(self):
+        queryset = super(FilterViewMixin, self).get_queryset()
+
+        assert self.serializer_class
```

Comment:
Do u really want to send a 500 error in case there is no serializer_class?

Reply:
I put a return queryset as a default first, but realized that this mixin makes no sense if you don't have a serializer class anyway, and the code was never used. The only reason an if would be appropriate is if a ViewSet changes serializer on runtime.. which seems like odd behaviour. What do you say?